example/server.js

```diff
@@ -3,44 +3,61 @@
 var Hapi = require('hapi');
 var pg = require('pg');
 var register = require('../lib/index.js');
+var tagsSystem = require('tags-system');
 
 function init (config, callback) {
   var server = new Hapi.Server();
   var pool = new pg.Pool(config.pg);
+  var optionsTags = {
+    tags: [],
+    categories: [],
+    pool: pool
+  };
 
   pool.on('error', function () {
     console.log('Pool error'); // eslint-disable-line
   });
   server.connection({ port: config.port });
 
-  server.register([{
-    register: register,
-    options: {
-      pool: pool,
-      reset: Boolean(process.env.RESET_TABLES_PEOPLE), // reset with content passed in the options, change the env to true and restart to add content
-      people: [],
-      organisations: [],
-      tags_organisations: []
-    }
-  }], function (err) {
-    if (err) {
-      return callback(err);
+  // register tags plugin which will create the tags table
+  // which is referenced in tags_organisations
+  server.register({
+    register: tagsSystem,
+    options: optionsTags
+  }, function (errorTags) {
+    if (errorTags) {
+      return callback(errorTags);
     }
 
-    server.route([
-      {
-        method: 'GET',
-        path: '/people',
-        handler: function (request, reply) {
-          request.pg.people.getAllPeople(function (error, response) { // eslint-disable-line
-            reply(response);
-          });
-        }
+    return server.register([{
+      register: register,
+      options: {
+        pool: pool,
+        reset: Boolean(process.env.RESET_TABLES_PEOPLE), // reset with content passed in the options, change the env to true and restart to add content
+        people: [],
+        organisations: [],
+        tags_organisations: []
```

Comment:
I'm wondering if it might be better to create a `peopleOptions`, at the top of the page, we specify 

```{ 
people: [ ],
organisations: [ ],
tags_organisations: [ ] 
}
```

so that we can see all the contents we are passing into the plugins easily? What are your thoughts on this?